doc/level1.rst

```diff
@@ -413,18 +413,34 @@ On message composition, an Autocrypt-capable MUA
 can decide whether to try to encrypt the new e-mail
 message.  Autocrypt provides a recommendation for the MUA.
 
-Any Autocrypt-capable MUA may have other means for making this
-decision outside of Autocrypt (see :doc:`other-crypto-interop`).
-Autocrypt provides a recommendation, but there is no
-requirement for Autocrypt-capable MUAs to follow this
-recommendation.
-
-That said, all Autocrypt-capable MUAs should be able to calculate
-the same Autocrypt recommendation.
-
-The Autocrypt recommendation depends on the recipient
-addresses of the draft message.  When the user changes the
-recipients, the Autocrypt recommendation may change.
+All Autocrypt-capable MUAs should be able to calculate the same
+Autocrypt recommendation.
+
+This recommendation algorithm provides sensible guidance that avoids
+many common problems, and Autocrypt-capable MUAs SHOULD follow the
+recommendation.  An implementation that deviates from the
+recommendation should do so on the basis of specific external evidence
+or knowledge, while carefully considering the impact of any variation,
+including:
+
+ * does it increase the chance of producing unexpectedly unreadable
+   mail (for either the sender or the recipient)?
+ * does it leak previously encrypted content in the clear?
+ * does it force the user to confront a choice they do not have the
+   information or knowledge to make safely?
+
+If an implementation deviates from the Autocrypt recommendation in a
+meaningful and useful way, the implementer should describe the
```

Comment:
i'd make this a SHOULD.

Reply:
SHOULD is usually intended for what implementations are expected to do.  This sentence is about what *implementers* are expected to do.  I'd rather not mix them up, but i don't feel strongly about it.  If you want to add a patch to this series that makes this a SHOULD i would be fine with it.